### PR TITLE
Final Evaluation Blog Post for Google Summer of Code

### PR DESCRIPTION
**Pull request by SidharthBansal**
No related image with this post

### CODE REVIEW

**Comment by davidevaleriani** [comment]
Waiting for an approval by @tacaswell or @story645 before merging

File: content/posts/GSoC_Coding_Phase_Blog_5/index.md
Comment (tacaswell) [inline content/posts/GSoC_Coding_Phase_Blog_5/index.md:27]: Add a bullet point for "how to intentionally change an image"?

**Review by tacaswell — APPROVED** [review]

**Comment by tacaswell** [comment]
Left one comment, but happy for this to be published as-is.

File: content/posts/GSoC_Coding_Phase_Blog_5/index.md
Comment (SidharthBansal) [reply content/posts/GSoC_Coding_Phase_Blog_5/index.md:27]: Done! Thanks

**Comment by SidharthBansal** [comment]
PR is ready to merge!!!

**Pull request merged**
